stumpy/mstump.py

```diff
@@ -1228,7 +1228,9 @@ def mstump(
         err = f"T is {T_A.ndim}-dimensional and must be at least 1-dimensional"
         raise ValueError(f"{err}")
 
-    core.check_window_size(m, max_size=min(T_A.shape[1], T_B.shape[1]))
+    # mstump currently supports self-join. Therefore, the argument `n` should be
```

Comment:
What is this comment for?

Reply:
IIRC, `mstump` only supports self-join and there was no study for a AB-join. IMHO, it does not even make sense to have AB-join for `mstump`. But, I might be wrong, and a paper might be published in the future to show `mstump` for AB-join. So, we may need to update this module to add such support. In that case, we probably need to avoid passing the argument `n` to the function `core.check_window_size` . 

The comment is to help our future selves to not forget that.

Reply:
Maybe "the argument `n=T_A.shape[1]` should be passed..."